packages/framework-integration-tests/src/index.ts

```diff
@@ -8,4 +8,4 @@ export {
   boosterTriggerScheduledCommand,
 } from '@boostercloud/framework-core'
 
-Booster.start()
+Booster.start(__dirname)
```

Comment:
Do we have now to change the project generator to have this as well?

Reply:
Yes @NickSeagull I already change the template. And another yes: this is a breaking change, but a really easy one to solve. I will let everybody know (I will create a release note for this)